src/manifestv3.json

```diff
@@ -11,19 +12,20 @@
   "content_scripts": [
     {
       "matches": ["http://*/*", "https://*/*"],
-      "js": ["content/content.js"]
+      "js": ["content/content.js"],
+      "all_frames": true
     },
     {
       "run_at": "document_start",
       "matches": ["http://*/*", "https://*/*"],
-      "js": ["contentStatic/contentStatic.js"]
+      "js": ["contentStatic/contentStatic.js"],
```

Comment:
We get CSP failure in Chrome at the moment. 
And Chrome doesn't allow lowering `content_security_policy.extension_pages` in any way (no nonce or SHA allowed).

Will need an approach with injectScript or web_accessible_resources I think.